spec/exporters/drug_stocks_report_exporter_spec.rb

```diff
@@ -2,193 +2,193 @@
 
 RSpec.describe DrugStocksReportExporter do
   around do |example|
-    I18n.with_locale(:en_IN) do
```

Comment:
I propose we pull this out into a separate PR since this one is pretty complex and this seems unrelated.

Reply:
Oh, if you view with whitespace ignored you'll see this is actually a very tiny change.  It just happened to change indentation throughtout the file.